includes/class-migration.php

```diff
@@ -802,9 +802,9 @@ private static function add_activitypub_capability() {
 	 * Add default notification settings.
 	 */
 	private static function add_notification_defaults() {
-		\add_option( 'activitypub_mailer_new_follower', '1' );
-		\add_option( 'activitypub_mailer_new_dm', '1' );
-		\add_option( 'activitypub_mailer_new_mention', '1' );
+		\add_option( 'activitypub_blog_user_mailer_new_follower', '1' );
```

Comment:
isn't it better to have a proper default instead of writing so many default entries?

Reply:
For options, sure! I updated these to stick with the precedence.

Reply:
Updated in [e208e0d](https://github.com/Automattic/wordpress-activitypub/pull/1586/commits/e208e0dcdbedcf9bdd29c89615e975f550352ca9)